clients/src/test/java/org/apache/kafka/clients/consumer/internals/ShareConsumeRequestManagerTest.java

```diff
@@ -457,9 +453,7 @@ public void testAcknowledgeOnClose() {
         shareConsumeRequestManager.fetch(Map.of(tip0, new NodeAcknowledgements(0, acknowledgements)));
 
         // Remaining acknowledgements sent with close().
-        Acknowledgements acknowledgements2 = Acknowledgements.empty();
-        acknowledgements2.add(2L, AcknowledgeType.ACCEPT);
-        acknowledgements2.add(3L, AcknowledgeType.REJECT);
+        Acknowledgements acknowledgements2 = getAcknowledgements(2, AcknowledgeType.ACCEPT, AcknowledgeType.ACCEPT, AcknowledgeType.REJECT);
```

Comment:
This was formerly ACCEPT, REJECT. Is the additional acknowledgement intentional?